### ISSUE TITLE
Mount plugin does not respect "icon size" directive

### ISSUE THREAD

**Issue by jleclanche** [lxde/lxqt#513]
![image](https://cloud.githubusercontent.com/assets/235410/6384155/54087f10-bd61-11e4-8478-817233579c56.png)

**Comment by intialonso** [comment]
In my system it changes as expected (changing the inconsise in the panel settings)..

**Comment by palinek** [comment]
I suppose this can be somehow connected to icons installed in system...i have this icons:
```
~$ find /usr/share/icons/ -name device-notifier\*
/usr/share/icons/oxygen/32x32/apps/device-notifier.png
/usr/share/icons/oxygen/128x128/apps/device-notifier.png
/usr/share/icons/oxygen/22x22/apps/device-notifier.png
```
when I'm changing icon size in panel configuration:
- 1 ~ 32px -> icon scales
- 33 ~ 80px -> icon is the same size (probably 32x32), but the button itself grows correctly
- >81px -> icon scales

**Comment by paulolieuthier** [comment]
Fixed with lxde/lxqt-panel@f47970ac58e9a5fc210dca4b42540a7f53fdb35b.
